.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+README.md
```

Comment:
- [ ] Kindly remove the README.md from the gitignore and add test.md instead as stated in the requirement.😉